tests/test_georaster.py

```diff
@@ -560,17 +560,44 @@ def test_split_bands(self):
 
         img = gr.Raster(datasets.get_path('landsat_RGB'))
 
-        red, green, blue = img.split_bands()
-
-        # Check that the red band is not equal to the full RGB data.
-        assert red != img
+        red, green, blue = img.split_bands(copy=False)
 
+        # Check that the shapes are correct.
         assert red.nbands == 1
+        assert red.data.shape[0] == 1
         assert img.nbands == 3
+        assert img.data.shape[0] == 3
+
+        # Extract only one band (then it will not return a list)
+        red2 = img.split_bands(copy=False, subset=0)
+
+        # Extract a subset with a list in a weird direction
+        blue2, green2 = img.split_bands(copy=False, subset=[2, 1])
+
+        # Check that the subset functionality works as expected.
+        assert np.array_equal(red.data.astype("float32"), red2.data.astype("float32"))
+        assert np.array_equal(blue.data.astype("float32"), blue2.data.astype("float32"))
+        assert np.array_equal(green.data.astype("float32"), green2.data.astype("float32"))
+
+        # Check that the red channel and the rgb data shares memory
+        assert np.shares_memory(red.data, img.data)
```

Comment:
I didn't know this function! So I guess that's the answer to my earlier question. If red.data is modified, so is img.data?

Reply:
Yes